csfieldguide/chapters/content/en/data-representation/sections/whats-the-big-picture.md

```diff
@@ -74,8 +74,8 @@ They are just using physical mechanisms such as high and low voltage, north or s
 # Bits
 
 The use of the two digits 0 and 1 is so common that some of the best known computer jargon is used for them.
-Since there are only two digits, the system is called binary.
-The short word for a "binary digit" is made by taking the first two letters and the last letter — a *{glossary-link term="bit"}bit{glossary-link end}* is just a digit that can have two values.
+Since there are only two digits, the system is called {glossary-link term="binary-number-system"}binary{glossary-link end}.
+The short word for a "binary digit" is made by taking the first two letters and the last letter &ndash; a *{glossary-link term="bit"}bit{glossary-link end}* is just a digit that can have two values.
```

Comment:
Why are some glossary terms in italics while others are not? Seems inconsistent. The existing text decoration should provide enough emphasis.

Reply:
It is the existing text decoration, no changes were made except to add glossary links

Reply:
(The italics could be removed though)